### PR TITLE
Made compile with removed alias this in Nullable

### PR DESCRIPTION
**Pull request by dukc**
Motivation: I want to update LDC in Nix package repository.

The `alias this` of `std.typecons.Nullable` has been removed and now explicit `.get`s are required.

### CODE REVIEW

File: BioD/bio/std/experimental/hts/bam/reader.d [hunk 1]
```diff
@@ -332,11 +332,14 @@ struct ProcessReadBlob {
...
+    assert(_read2.get.is_mapped,"Trying to get consumed bases on an unmapped read"); // BAM spec
     assert(!read_name.isNull,"Trying to get CIGAR on RNAME is '*'"); // BAM spec
-    auto raw = cast(uint[]) _read2.raw_cigar();
+    auto raw = cast(uint[]) _read2.get.raw_cigar();
     if (raw.length==1 && raw[0] == '*')
-      return consumed_reference_bases2; // null
+      // The existing comment here says "null", but function return type
+      // forces reading the value anyway. Bug?
+      // -Nix packager
+      return consumed_reference_bases2.get; // null
```
Comment (dukc) [inline BioD/bio/std/experimental/hts/bam/reader.d:342]: I also spotted a potential bug while doing this and left a comment accordingly

**Review by PetarKirov — APPROVED** [review]

**Comment by PetarKirov** [comment]
Ping @pjotrp

**Comment by pjotrp** [comment]
Thank you, lgtm.

**Comment by dukc** [comment]
@PetarKirov Huh? How did you end up here 😀?

**Pull request merged**